$OpenBSD: patch-phonon_CMakeLists_txt,v 1.3 2015/07/06 23:54:04 zhuk Exp $
Revert upstream commit 457492e97dd1435021cff15d4fbb8f1fd7be9b24, it breaks
due to changing "-I/usr/local/include/kde4" to
"-isystem /usr/local/include/kde4".
--- phonon/CMakeLists.txt.orig	Tue Jul  7 02:46:03 2015
+++ phonon/CMakeLists.txt	Tue Jul  7 02:46:41 2015
@@ -172,8 +172,6 @@ set_target_properties(${PHONON_LIB_SONAME} PROPERTIES
 # This reduces inter-package dependencies and makes it easier to remove
 # dependencies of shared libraries without breaking binary compatibility.
 
-target_include_directories(${PHONON_LIB_SONAME}
-    INTERFACE "$<INSTALL_INTERFACE:${INCLUDE_INSTALL_DIR}>")
 
 install(TARGETS ${PHONON_LIB_SONAME} EXPORT PhononLibs ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
